src/infra/docs/dev-desktop.md

```diff
@@ -162,6 +162,19 @@ In summary:
    rust-analyzer)
 7. Use VS Code to run or debug the code remotely
 
+## How to request new packages and tools
+
+If you need a specific tool or package that is not installed on the dev
+desktops, you can request its installation by opening a pull request against
+the [rust-lang/simpleinfra] repository. The Ansible role [`dev-desktop`](https://github.com/rust-lang/simpleinfra/blob/master/ansible/roles/dev-desktop)
```

Comment:
I think it's best to link to a version or commit SHA. The master branch isn't guaranteed to reflect the current state for eternity and the structure may change based on a design decision.

Reply:
I think I disagree. I think this lint should always be up to date, since folks are going to check out the role and its configurations. If it breaks, then it needs to be updated 😺

Reply:
I feel the same. I rather have this point to the latest docs and break if the file gets moved than have people try to modify an outdated file. And historically, file paths inside the `/ansible` directory have been extremely stable. So I don't think this will be an issue.